src/main/java/net/openhft/chronicle/bytes/internal/HeapBytesStore.java

```diff
@@ -139,7 +140,6 @@ protected void performRelease() {
         return capacity;
     }
 
-    @NotNull
```

Comment:
I'm still unclear on how these annotations are interpreted, do we need to add a @Nullable annotation here or do we assume anything not @NotNull is @Nullable? (and if that's the case, why have the annotation on the constructor arg?)